vipps-report-api-faq.md

```diff
@@ -33,6 +34,49 @@ We are still quite early in the planning and design phase and will use the
 draft documentation here to discuss with potential users of the API
 and make sure we are developing the right solution.
 
+## Which API keys give access to the API?
+
+In short: The following API keys give access to the Report API:
+* The merchant's own API keys (the same API keys that are used to make payments)
+* Partner keys (the same API keys that are used to make payments)
+* Accounting partners' API keys (after the merchant has given the accounting partner access)
```

Comment:
I don't understand Accounting partners' API keys... are they:  (the API keys provided to the accounting partner after the merchant has given them access)?